### ISSUE TITLE
TOR requires google bot verification

### ISSUE THREAD

**Issue by notkyllian** [rndinfosecguy/Scavenger#2]

**Comment by rndinfosecguy** [comment]
Hey! Thanks for letting me know about this issue. Just tested it a bit and I get this behaviour from time to time to when using the TOR version.
However, for me this just occurs for specific tor cycles. 
If you have this problem regardless of your cycle, please give me some more info and let me know :)
Keep in mind that I recommend to use "fast TOR cycles" to minimize that problem 

<img width="910" alt="Screenshot 2019-12-02 at 21 22 25" src="https://user-images.githubusercontent.com/37249167/69992539-2920cc80-154a-11ea-87e9-c3036a4db5a8.png">

Let me know if this fixes your issue or if you need further assistent

**Comment by notkyllian** [comment]
Do you have discord? I have that value set in my torrc file. I use it in the startup of my tor  since I'm on windows

**Comment by notkyllian** [comment]
![image](https://user-images.githubusercontent.com/16906265/69996550-933d6f80-1552-11ea-8abd-75131223fa2a.png)

**Comment by rndinfosecguy** [comment]
Ah. I never tested it on Windows.
Let me test it and I will give you some feedback then. :)

**Comment by omnieusza** [comment]
Hi There,
I am also using the TOR version(for now) and also been getting the Captcha verification.
I have the MaxCircuitDirtiness set (I am on Kali) but still getting the Captcha.
2 things I have tried so far, adjusting the user agent in P_bot_TOR.py to match a full Tor Browser, didn't work.
I also routed my normal firefox browser through Tor as a Socks5 proxy, the pages then opened and worked as expected. 

I suspected it might have had something to do with browser detection, requests via Tor using normal Firefox work fine(to Pastebin) however when running through python, you are presented with the Captcha. 

Hope this helps
